keyboards/wuque/epiphany70/keymaps/via/keymap.c

```diff
@@ -0,0 +1,48 @@
+        KC_PAUS,    KC_LCTL, KC_LGUI, KC_LALT,                   KC_SPC,  KC_SPC,  KC_SPC,                    KC_RALT, KC_LGUI, MO(1),   KC_RCTL,             KC_LEFT, KC_DOWN, KC_RGHT
+    ),
+    [1] = LAYOUT_all(
+                    _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,    _______, _______, QK_BOOT,
+        _______,    _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,             _______, _______, _______,
+        _______,    _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,
+        _______,    _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,                      _______,
+        _______,    _______, _______, _______,                   _______, _______, _______,                   _______, _______, _______, _______,             _______, _______, _______
+    ),
+    [2] = LAYOUT_all(
+                    _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,    _______, _______, _______,
+        _______,    _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,             _______, _______, _______,
+        _______,    _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,
+        _______,    _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,                      _______,
+        _______,    _______, _______, _______,                   _______, _______, _______,                   _______, _______, _______, _______,             _______, _______, _______
+    ),
+    [3] = LAYOUT_all(
+                    _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,    _______, _______, _______,
+        _______,    _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,             _______, _______, _______,
+        _______,    _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,
+        _______,    _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,                      _______,
+        _______,    _______, _______, _______,                   _______, _______, _______,                   _______, _______, _______, _______,             _______, _______, _______
+    ),
```

Comment:
remove non-functional layers. VIA defaults to 4 layers regardless if 4 layers are specified, or not, in keymap
```suggestion
    )
```